updater/spec/support/dummy_pkg_helpers.rb

```diff
@@ -90,7 +90,7 @@ def deprecated?
     sig { override.returns(T::Boolean) }
     def unsupported?
       # Check if the feature flag for Bundler v1 unsupported error is enabled.
```

Comment:
this comment seems no longer relevant after this code change?

Reply:
removed.